algopy_testing/src/algopy/op.py

```diff
@@ -152,3 +156,15 @@ def ecdsa_pk_decompress(v: ECDSA, a: Bytes | bytes, /) -> tuple[Bytes, Bytes]:
         return Bytes(pubkey_x.to_bytes(32, byteorder="big")), Bytes(
             pubkey_y.to_bytes(32, byteorder="big")
         )
+
+
+def vrf_verify(
+    s: VrfVerify,  # noqa: ARG001
```

Comment:
since these arguments are positional only, can prefix the arg names with _ and then remove the `# noqa`

Reply:
@daniel-makerx this wouldn't be an issue with the fact that the argument names will be different from the ones in the stub definition? The '_' in front of args wouldn't affect anything?